### PR TITLE
docs: update contrib doc

### PR DESCRIPTION
**Pull request by ra0x3**
@deekerno  @hrryndrsn 

Now that the team is growing, we're standardizing our contribution process a bit more, checkout the updates. Feel free to ask any questions 👌

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -113,27 +113,46 @@ This is a rough outline of what a contributor's workflow looks like:
...
+
 Thanks for your contributions!
 
 ## Finding something to work on
 
 For beginners, we have prepared many suitable tasks for you. Checkout our [Good First Issues](https://github.com/FuelLabs/fuel-indexer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for a list.
 
-If you are planning something big, for example, relates to multiple components or changes current behaviors, make sure to open an issue to discuss with us before continuing.
\ No newline at end of file
+If you are planning something big, for example, relates to multiple components or changes current behaviors, make sure to open an issue to discuss with us before continuing.
```
Comment (deekerno) [inline docs/CONTRIBUTING.md:158]: Consider:
> If you are planning something that changes current behaviors or relates to multiple components, ...

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -113,27 +113,46 @@ This is a rough outline of what a contributor's workflow looks like:
...
+  - Give your branch a name related to the work you're doing
 - Write code, add test cases, and commit your work.
 - Run tests and make sure all tests pass.
+- Your commit message should be formatted as `[commit type]: [short commit blurb]`
+  - Examples:
+    - If you fixed a bug, your message is `fix: sqlite database locking issue`
+    - If you added new functionality, your message would be `enhancement: i add
+        something super cool`
+    - If you just did a chore your message is: `chore: i did somthing not fun`
+    - So on and so forth
```
Comment (deekerno) [inline docs/CONTRIBUTING.md:125]: Nit: Remove this line, and un-indent "Keeping..." as they're not examples.

**Review by deekerno — CHANGES_REQUESTED** [review]

**Review by hrryndrsn — DISMISSED** [review]
Looks good 👍

**Review by deekerno — APPROVED** [review]
👍🏽

**Pull request merged**